scripts/pre_push_hook.py

```diff
@@ -60,7 +60,9 @@
 FILE_DIR: Final = os.path.abspath(os.path.dirname(__file__))
 OPPIA_DIR: Final = os.path.join(FILE_DIR, os.pardir, os.pardir)
 LINTER_FILE_FLAG: Final = '--files'
-PYTHON_CMD: Final = 'python'
+# Path to currently running python interpreter
+# It is needed to resolve python version conflict in docker
+PYTHON_CMD: Final = sys.executable
```

Comment:
Need review about usage of sys.executable is okay instead of python. sys.executable returns path of current running interpreter